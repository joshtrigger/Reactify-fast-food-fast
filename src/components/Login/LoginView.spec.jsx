import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import { LoginView } from "./LoginView";

configure({adapter: new Adapter()})

let wrapper;
let props = { onChange: jest.fn(), onSubmit: jest.fn(), message: "" };

describe('test login component', () => {
  beforeEach(()=>{
    wrapper = shallow(<LoginView {...props}/>)
  })
  it('match snapshots', ()=>{
    expect(wrapper).toMatchSnapshot()
  })
})

