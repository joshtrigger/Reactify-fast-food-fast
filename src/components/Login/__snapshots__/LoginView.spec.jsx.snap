// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test login component match snapshots 1`] = `ShallowWrapper {}`;
